modules/lvms-provisioning-storage-using-logical-volume-manager-operator.adoc

```diff
@@ -4,67 +4,75 @@
-= Provisioning storage using {lvms}
+= Provisioning storage
 
-You can provision persistent volume claims (PVCs) using the storage class that is created during the Operator installation. You can provision block and file PVCs, however, the storage is allocated only when a pod that uses the PVC is created.
+After you have created the LVM volume groups using the `LVMCluster` custom resource (CR), you can provision the storage by creating persistent volume claims (PVCs).
 
-[NOTE]
-====
-{lvms} provisions PVCs in units of 1 GiB. The requested storage is rounded up to the nearest GiB.
-====
+To create a PVC, you must create a `PersistentVolumeClaim` object.
+
+.Prerequisites
+
+* You have created an `LVMCluster` CR.
 
 .Procedure
 
-. Identify the `StorageClass` that is created when {lvms} is deployed.
-+
-The `StorageClass` name is in the format, `lvms-<device-class-name>`.
-The `device-class-name` is the name of the device class that you provided in the `LVMCluster` of the `Policy` YAML.
-For example, if the `deviceClass` is called `vg1`, then the `storageClass` name is `lvms-vg1`.
-+
-The `volumeBindingMode` of the storage class is set to `WaitForFirstConsumer`.
+. Log in to the OpenShift CLI (`oc`).
 
-. To create a PVC where the application requires storage, save the following YAML to a file with a name such as `pvc.yaml`.
+. Create a `PersistentVolumeClaim` object:
 +
-.Example YAML to create a PVC
+.Example `PersistentVolumeClaim` object
 [source,yaml]
 ----
-# block pvc
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: lvm-block-1
-  namespace: default
-spec:
-  accessModes:
-    - ReadWriteOnce
-  volumeMode: Block
-  resources:
-    requests:
-      storage: 10Gi
-  storageClassName: lvms-vg1
----
-# file pvc
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: lvm-file-1
+  name: lvm-block-1 <1>
   namespace: default
 spec:
   accessModes:
     - ReadWriteOnce
-  volumeMode: Filesystem
+  volumeMode: Block <2>
   resources:
     requests:
-      storage: 10Gi
-  storageClassName: lvms-vg1
+      storage: 10Gi <3>
+  storageClassName: lvms-vg1 <4>
 ----
+<1> Specify a name for the PVC.
+<2> To create a block PVC, set this field to `Block`. To create a file PVC, set this field to `Filesystem`.
+<3> Specify the storage size. {lvms-first} provisions PVCs in units of 1 GiB (gibibytes). The requested storage is rounded up to the nearest GiB. The total storage size you can provision is limited by the size of the LVM thin pool and the overprovisioning factor.
+<4> The value of the `storageClassName` field must be in the format `lvms-<device_class_name>` where `<device_class_name>` is the value of the `deviceClasses.name` field in the `LVMCluster` CR.
+For example, if the `deviceClasses.name` field is set to `vg1`, you must set the `storageClassName` field to `lvms-vg1`.
++
+[NOTE]
+====
+The `volumeBindingMode` field of the storage class is set to `WaitForFirstConsumer`.
```

Comment:
Is this something they need to set somewhere? Might need a bit more detail to follow through if so.

Reply:
@jeana-redhat 
The storage class referred to here is automatically created by LVM Storage when users create the "LVMCluster CR".

We are specifying that the `WaitForFirstConsumer` field will be set to a value by default. The purpose of adding this line is that this configuration is the reason behind the following behavior: (which is added as a note at the end of this proc)

"The created PVCs remain in `Pending` state until you deploy the workloads that use them."

Reply:
Sounds good to me! I have previously encountered a procedure that required the user to set that value, but if this one is automatic sounds like they are all set. Thanks for explaining!